### PR TITLE
Tag GLVisualize.jl v0.3.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaGL/GLVisualize.jl](https://github.com/JuliaGL/GLVisualize.jl)
Release: [v0.3.0](https://github.com/JuliaGL/GLVisualize.jl/releases/tag/v0.3.0)
Diff: [vs v0.2.2](https://github.com/JuliaGL/GLVisualize.jl/compare/26b34816b7762f12bc2f885fa37f1aa6d9717c04...73b12c5a33b8a87e1ad20b28388a2a6eaa4a54cc)
`requires` vs v0.2.2: 
```diff
--- 0.2.2/requires
+++ 0.3.0/requires
@@ -1,27 +1,36 @@
-julia 0.5
+# I plan to release an 0.6 only version after this,
+# so we need to cap everything that relies on StaticArrays (that's making it 0.6 only)
 
-GLWindow 0.3
-GLAbstraction 0.3.1
-GeometryTypes 0.2
-Compat 0.9.5
+# this shouldn't include 0.6!
+julia 0.5 0.6.0
+
+# Packages that soon will rely on StaticArrays
+GLWindow 0.3.3 0.4
+GLAbstraction 0.3.2 0.4
+GeometryTypes 0.2 0.3
+Packing 0.0.4 0.1
+Reactive 0.3.0 0.3.7 0.5-
+
+# packages with stable interface
 
 GLFW
 ModernGL
 FixedSizeArrays
-Reactive
 
 FixedPointNumbers
 ColorVectorSpace
 ColorTypes
 Colors
-
-Images
-Quaternions
-FileIO
-MeshIO
-
 SignedDistanceFields
 FreeType
 Iterators
-Packing
+FileIO
+MeshIO
 UnicodeFun
+Compat 0.18
+Quaternions
+
+# We went through extra length to be compatible to most Images versions
+Images
+AxisArrays
+ImageAxes
```
cc: @SimonDanisch

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

**Comment by SimonDanisch** [comment]
If this require looks good, I guess the next step will be to update older releases to use a similar require!

File: GLVisualize/versions/0.3.0/requires [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# this shouldn't include 0.6!
+julia 0.5 0.6.0
+
+# Packages that soon will rely on StaticArrays
+GLWindow 0.3.3 0.4
+GLAbstraction 0.3.2 0.4
+GeometryTypes 0.2 0.3
+Packing 0.0.4 0.1
+Reactive 0.3.0 0.3.7 0.5-
```
Comment (tkelman) [inline GLVisualize/versions/0.3.0/requires]: 0.4 doesn't work?

Comment (SimonDanisch) [inline GLVisualize/versions/0.3.0/requires]: Yes! There were still some bugs in it...

**Comment by tkelman** [comment]
examples/plots/3dplots.jl depends on Plots and gets run by the tests but Plots isn't in either REQUIRE or test/REQUIRE

**Comment by SimonDanisch** [comment]
The example runner checks if Plots is present or not and depending on it doesn't include the Plot tests.

**Comment by tkelman** [comment]
> next step will be to update older releases to use a similar require

yes, all these upper bounds won't be effective unless they're present on the past versions

**Comment by SimonDanisch** [comment]
yeah I'd like to do this after merging this!

**Comment by tkelman** [comment]
> yeah I'd like to do this after merging this!

Never happened? None of the upper bounds have been accomplishing what you put them there for because of this. It just means GLVisualize could be downgraded to 0.2.x to upgrade any of the bounded dependencies.

**Pull request merged**